package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
+    "build:tailwind": "tailwindcss -i public/input.css -o public/output.css",
```

Comment:
if you're doing build:tailwind every time something changes in the css, we may want a script command that combines build:dev with build:tailwind

Reply:
something like this? `"build:devtail": "npm run build:dev && npm run build:tailwind"`

Reply:
yes exactly

Reply:
done and thanks!